implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v1/DynatraceNamingConvention.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micrometer.dynatrace;
+package io.micrometer.dynatrace.v1;
```

Comment:
This is a breaking change, and based on @shakuzen's comment  (users can extend it and implement their own logic) we should not break this in a minor release.
What we can do instead is something similar that we did with the meter registry and hide the v1/v2 logic behind this class: having a v1 NamingConvention and a v2 NamingConvention and based on a ctor param set the right one (delegate), the default is v1.

Reply:
Moved it back up a level and added a delegate as suggested.